include/swift/AST/TypeCheckRequests.h

```diff
@@ -4423,6 +4423,25 @@ class SerializeAttrGenericSignatureRequest
   void cacheResult(GenericSignature signature) const;
 };
 
+class IsFunctionBodySkippedRequest
+    : public SimpleRequest<IsFunctionBodySkippedRequest,
+                           bool(const AbstractFunctionDecl *),
+                           RequestFlags::Cached> {
```

Comment:
Looks like this should be `RequestFlags::SeparatelyCached`

Reply:
Thanks for the catch! What does specifying this correctly do? I've apparently made the mistake before and I haven't noticed any consequences which is a little surprising.